public/js/reducers/breakpoints.js

```diff
@@ -146,11 +156,26 @@ function getBreakpointsForSource(state: OuterState, sourceId: string) {
   });
 }
 
+function getBreakpointsDisabled(state: OuterState) {
+  return state.breakpoints.get("breakpointsDisabled");
+}
+
+function getBreakpointsLoading(state: OuterState) {
+  const breakpoints = getBreakpoints(state);
+  const isLoading = !!breakpoints.valueSeq()
+                    .filter(bp => bp.loading)
+                    .first();
```

Comment:
You can just check `size` or I think immutable does have an `empty` predicate on collections